rhtap.groovy

```diff
@@ -73,3 +73,11 @@ def gather_deploy_images( ) {
 def verify_enterprise_contract( ) {
     run_script ('verify-enterprise-contract.sh')
 }
+
+def gather_images_to_upload_sbom( ) {
+    run_script ('gather-images-to-upload-sbom.sh')
```

Comment:
I you might need something similar to https://github.com/redhat-appstudio/tssc-dev-multi-ci/pull/43 to make sure the gather-deploy-images is present.

Reply:
Good point. Currently it works, but only because gather-deploy-images is also used as a standalone step before the gather-images-to-upload-sbom step. Will fix